ui/pages/confirmations/components/confirm/pluggable-section/pluggable-section.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ReactComponentLike } from 'prop-types';
+import { useSelector } from 'react-redux';
+
+import { currentConfirmationSelector } from '../../../selectors';
+
+// Components to be plugged into confirmation page can be added to the array below
+const PluggedInSections: ReactComponentLike[] = [];
+
+const PluggableSection = () => {
+  const currentConfirmation = useSelector(currentConfirmationSelector);
+
+  return (
+    <>
+      {PluggedInSections.map((Section) => (
+        <Section confirmation={currentConfirmation} />
```

Comment:
Do these need a `key` property since we're rendering a dynamic list?

Reply:
good catch, let me add that.

Reply:
Done 👍